usr/src/uts/common/os/kmem.c

```diff
@@ -2186,9 +2186,10 @@ kmem_depot_ws_zero(kmem_cache_t *cp)
 }
 
 /*
- * Setting it this high (1GB) casues the explicit kpreempt to have
- * very little effect.  Setting it to 1MB causes us to call
- * kpreempt() approximately every 2 milliseconds.
+ * Setting it this high (1GB) causes the explicit kpreempt() to have very
+ * little effect. To make calls to kpreempt() more frequent, use a smaller
+ * value. A few megabytes is usually small enough that it will run hundreds of
+ * times per second.
  */
 size_t kmem_reap_preempt_bytes = 1024 * 1024 * 1024;
```

Comment:
I'd recommend turning this "on" by default (i.e. setting a lower value), as we have in more recent DxOS.  (@dankimmel see commit bb204e0997051e7558011aa8607ceed5975c607a)